Doc/using/cmdline.rst

```diff
@@ -1091,6 +1092,19 @@ conflict.
 
    .. versionadded:: 3.13
 
+.. envvar:: PYTHON_FROZEN_MODULES
+
+   If this variable is set to ``on`` or ``off``, it determines whether or not
+   frozen modules are ignored by the import machinery.  A value of "on" means
+   they get imported and "off" means they are ignored.  The default is "on"
+   for non-debug builds (the normal case) and "off" for debug builds.
+   Note that the "importlib_bootstrap" and "importlib_bootstrap_external"
+   frozen modules are always used, even if this flag is set to "off".
```

Comment:
```suggestion
   If this variable is set to ``on`` or ``off``, it determines whether or not
   frozen modules are ignored by the import machinery.  A value of ``on`` means
   they get imported and ``off`` means they are ignored.  The default is ``on``
   for non-debug builds (the normal case) and ``off`` for debug builds.
   Note that the :mod:`!importlib_bootstrap` and :mod:`!importlib_bootstrap_external`
   frozen modules are always used, even if this flag is set to ``off``.
```

And we can apply the same formatting to the ``-X frozen_modules`` section above.